### PR TITLE
Consistently use relative imports

### PR DESCRIPTION
**Pull request by stefanv**

### CODE REVIEW

**Comment by blink1073** [comment]
This looks good to me, waiting for CI thumbs up.

**Comment by vighneshbirodkar** [comment]
Just a suggestion : Should we mention this in our contributing guide ?

**Comment by stefanv** [comment]
Updated

**Comment by JDWarner** [comment]
@stefanv tagging this for the sprint, as it seems mostly done pending Travis. I restarted the two failing builds, which appeared to be Travis unclosed file issues.

File: skimage/_shared/testing.py [hunk 1]
```diff
@@ -181,15 +181,19 @@ def mono_check(plugin, fmt='png'):
 def setup_test():
     """Default package level setup routine for skimage tests.
 
-    Import packages known to raise errors, and then
+    Import packages known to raise warnings, and then
     force warnings to raise errors.
-    Set a random seed
+
+    Also set a random seed.
```
Comment (stefanv) [inline skimage/_shared/testing.py]: Set the random seed to '0'.  This sentence can easily be interpreted that the seed gets set randomly.

**Comment by stefanv** [comment]
Nice, I like how I just reviewed my own PR--inception.

**Comment by stefanv** [comment]
Rebased on master

**Pull request merged**